README.md

```diff
@@ -85,8 +87,11 @@ The encoded bytes of the struct from above example will look like:
  00000040  0c 30 31 30 2d 31 32 33  34 35 36 37 38           |.010-12345678|
 ```
 
-#### Struct filed name decoding
-Hessian decoder will compare all filed's name of struct until it matches the correct name, the order of matching rules is:
+#### Decoding filed name 
+
+Hessian decoder find the correct target field though comparing all filed names of struct one by one until matching.
```

Comment:
Hessian decoder finds the correct target field through comparing all filed names of struct one by one until matching.